src/client/app/components/conversion/SplitConversionSegmentModalComponent.tsx

```diff
@@ -0,0 +1,233 @@
+ * Renders a modal that allows the user to split a conversion segment in either direction.
+ * The new segment starts with neutral values (slope=0, intercept=0, no pattern)
+ * so the admin must provide or confirm values before saving.
+ * @param props The properties for the component
+ * @returns A modal component for splitting a conversion segment
+ */
+export default function SplitConversionSegmentModalComponent(props: SplitConversionSegmentModalComponentProps): React.ReactElement {
+	const translate = useTranslate();
+	const defaultSegmentValues = selectDefaultSplitConversionSegmentValues(props.segment);
+	const [newSegment, setNewSegment] = React.useState<SplitConversionSegmentPayload>(defaultSegmentValues);
+	const [showWarningModal, setShowWarningModal] = React.useState(false);
+	const [fieldError, setFieldError] = React.useState<string>('');
+
+	const [splitEarlier] = conversionSegmentsApi.useSplitConversionSegmentEarlierMutation();
+	const [splitLater] = conversionSegmentsApi.useSplitConversionSegmentLaterMutation();
+
+	const handleNumberChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+		setNewSegment(prev => ({
+			...prev,
+			[e.target.name]: Number(e.target.value)
+		}));
+	};
+
+	const handleStringChange = (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+		setNewSegment(prev => ({
+			...prev,
+			[e.target.name]: e.target.value
+		}));
+	};
+
+	const handlePatternChange = (e: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>) => {
+		const selectedPattern = Number(e.target.value);
+		setNewSegment(prev => {
+			if (selectedPattern === -99) {
+				return {
+					...prev,
+					newWeekPatternsId: -99,
+					newSlope: 0,
+					newIntercept: 0
+				};
+			} else {
+				return {
+					...prev,
+					newWeekPatternsId: selectedPattern
+				};
+			}
+		});
+	};
+
+	const handleSplitTimeChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+		setNewSegment(prev => ({
+			...prev,
+			splitTime: e.target.value
+		}));
+	};
+
+
+	const doMutation = async () => {
+		const payload: SplitConversionSegmentPayload = {
+			...newSegment,
+			sourceId: props.segment.sourceId,
+			destinationId: props.segment.destinationId,
+			startTime: props.segment.startTime,
+			endTime: props.segment.endTime,
+			splitTime: newSegment.splitTime,
+			newWeekPatternsId: newSegment.newWeekPatternsId,
+			newNote: newSegment.newNote || ''
+		};
+
+		try {
+			if (props.direction === 'earlier') {
+				await splitEarlier(payload).unwrap();
+			} else {
+				await splitLater(payload).unwrap();
+			}
+			showSuccessNotification(translate('conversion.segment.split.success'));
+			props.handleClose();
+		} catch (error) {
+			showErrorNotification(translate('conversion.segment.split.error'));
+		}
+	};
+
+	const handleSplitSegment = async () => {
+		const splitDatetime = moment(newSegment.splitTime, 'YYYY-MM-DD HH:mm:ss', true);
+
+		if (!splitDatetime.isValid()) {
+			setFieldError(translate('conversion.error.datetime.invalid'));
```

Comment:
I noticed that after this notice appears it says even after the user types in new info. Maybe it should clear after any typing?